utils/output_encoders.go

```diff
@@ -204,15 +204,22 @@ var GDALTypes = map[string]C.GDALDataType{"Unkown": 0, "Byte": 1, "UInt16": 2, "
 
 func EncodeGdal(format string, rs []Raster, geot []float64, epsg int) ([]byte, error) {
 	var driverName string
+	var driverOptions []*C.char
 	switch strings.ToLower(format) {
 	case "geotiff":
 		driverName = "GTiff"
+		driverOptions = append(driverOptions, C.CString("COMPRESS=LZW"))
 	case "netcdf":
 		driverName = "netCDF"
+		driverOptions = append(driverOptions, C.CString("COMPRESS=DEFLATE"))
+		driverOptions = append(driverOptions, C.CString("ZLEVEL=6"))
 	default:
 		return []byte{}, fmt.Errorf("Unsupported encoding format: %v", format)
 	}
 
+	// NULL pointer is used to terminate the point array by gdal
```

Comment:
Suggest this comment: `// Terminate the array of C strings for GDAL`, if this comment is even needed.